UQ-eLIPSE/crucible-components
# feature/VETHUB-111

_Please ignore the large amount of commits. I accidentally merged this branch with the release package but I've now reverted it back. The commits from the merge are not applied._

```gitattributes
git fetch
git checkout feature/VETHUB-111
```
To test, running `yarn test` should pass all tests.

To test in client
```gitattributes
git checkout release-package 
git pull
```
Head to `localhost:5173`
Make sure you've completed all the steps outlined in https://github.com/UQ-eLIPSE/crucible/pull/61
Go to the smart quiz section and choose the second one..
The api data retrieved from the backend should be reflected here in the client.

Future steps: Delete all the static files and functions in the `DataAccessLayer` we're not using and implement _type guards_
